book/common/definitions.tex

```diff
@@ -611,7 +611,7 @@
 
 \begin{SaveDefinition}[key=LinearTransformationinaBasis, title={Linear Transformation in a Basis}]
 	Let $\mathcal T:\R^n\to\R^n$ be a linear transformation and let $\mathcal B$ be a
-	basis for $\R^n$. The \emph{matrix for $\mathcal T$ with respect to $\mathcal B$}\index[definitions]{Linear transformation! matrix representation of}, notated
```

Comment:
I removed this index entry because it's already accounted for a few lines below as `\index[definitions]{Linear transformation!representation in a basis}`